src/components/custom-aggrid/custom-aggrid-header.jsx

```diff
@@ -70,17 +70,18 @@ const styles = {
  */
 export function DisplayRounding({ decimalAfterDot }) {
     const intl = useIntl();
+    const roundedTo1 = decimalAfterDot === 0;
     const displayAsPower10 = decimalAfterDot > 4;
     const baseMessage =
         intl.formatMessage({
-            id: 'filter.rounded',
+            id: roundedTo1 ? 'filter.roundedToOne' : 'filter.rounded',
         }) + ' ';
 
     const decimalAfterDotStr = -decimalAfterDot;
     return (
         <FormHelperText>
             {baseMessage}
-            {displayAsPower10 ? (
+            {roundedTo1 ? null : displayAsPower10 ? (
```

Comment:
Sonar displays a warning on this double ternary : 

you should change code to : 

    let rounded = null;
    if (!roundedTo1) {
        rounded = displayAsPower10 ? (
            <>
                10
                <Box component="span" sx={styles.exponent}>
                    {decimalAfterDotStr}
                </Box>
            </>
        ) : (
            1 / Math.pow(10, decimalAfterDot)
        );
    }
    return (
        <FormHelperText>
            {baseMessage}
            {rounded}
        </FormHelperText>
    );

Reply:
Done as per your suggestion.

I only renamed "rounded" to "roundingPrecision".